ppl/detail/RelatedConns.tsx

```diff
@@ -0,0 +1,57 @@
+import {Data, Name, Value} from "app/core/Data"
+import {BrimEvent, BrimEventInterface} from "ppl/detail/models/BrimEvent"
+import React, {memo, useMemo} from "react"
+import brim from "src/js/brim"
+import EventTimeline from "./EventTimeline"
+import {TableWrap, ChartWrap, Caption} from "app/detail/Shared"
+import formatDur from "./util/formatDur"
+import PanelHeading from "app/detail/PanelHeading"
+import Panel from "app/detail/Panel"
+import {zng} from "zealot"
+import EventLimit from "./EventLimit"
+import useSearch from "app/core/hooks/useSearch"
+import firstLast from "./util/firstLast"
+import zql from "src/js/zql"
+
+type Props = {
+  record: zng.Record
+}
+
+const LIMIT = 100
+const getQuery = (r: zng.Record, limit?: number) => {
+  const cid = r.get("community_id")
+  const base = zql`_path=conn community_id=${cid} | sort ts`
+  return limit ? `${base} | head ${limit}` : base
+}
+
+export default memo(function RelatedConns({record}: Props) {
+  const [records, isFetching] = useSearch(getQuery(record, LIMIT), [record])
+  const events = useMemo(() => records.map(BrimEvent.build), [records])
+  const [first, last] = firstLast<BrimEventInterface>(events)
+  const data = [
+    ["Count", events.length],
+    ["First ts", first ? brim.time(first.getTime()).format() : "Not available"],
+    ["Last ts", last ? brim.time(last.getTime()).format() : "Not available"],
+    ["Duration", formatDur(first?.getTime(), last?.getTime())]
+  ]
+  return (
+    <section>
+      <PanelHeading isLoading={isFetching}>Related Connections</PanelHeading>
+      <Panel>
+        <ChartWrap>
+          <EventTimeline events={events} />
+          <EventLimit query={getQuery(record)} count={events.length} />
+        </ChartWrap>
+        <TableWrap>
+          {data.map(([name, value]) => (
+            <Data key={name}>
+              <Name>{name}</Name>
+              <Value>{value}</Value>
+            </Data>
+          ))}
+        </TableWrap>
+      </Panel>
+      <Caption>Populated by community_id</Caption>
+    </section>
+  )
+})
```

Comment:
This component render's this piece.

<img width="304" alt="image" src="https://user-images.githubusercontent.com/3460638/103564618-b2a07d80-4e73-11eb-8053-b673a7e85293.png">